Add emails and urls to HTML output (#2539)
nexB/scancode-toolkit: CHANGELOG.rst

   that contains each package instace that can be aggregating data from
   multiple manifests for a single package instance.

>  - The data structure for HTML output has been a change for emails and URLs under Infos. 
>    Now HTML template will output holders, authors, emails, and URLs information into 
>    separate tables for each like licenses and copyrights.
> 

Reviewer: ```suggestion
 - The data structure for HTML output has been changed to include emails and urls under the 
   "infos" object. Now HTML template will output holders, authors, emails, and 
   urls into separate tables like "licenses" and "copyrights".
```
